saplings/circuits/src/index.js

```diff
@@ -22,5 +22,7 @@ import './index.css';
 import App from './App';
 
 registerApp(domNode => {
-  ReactDOM.render(<App />, domNode);
+  if (window.location.pathname === '/circuits') {
```

Comment:
This maybe should not be a full equality. If I try to go to a URL like `/circuits/propose` (which will be implemented in a future PR), I'm redirect to the login screen. However after the login screen I'm redirected to `/circuits/propose` and the app does not load.

Reply:
The profile sapling has a special case because it is a config sapling, but also renders a UI.

For this fix, I think it will require an update to CanopyJS and/or the register-login sapling. We could choose to not render any saplings unless there is a user logged in (could be a good solution unless you have a canopy app that isnt using Biome at all) or fix whatever is causing the register-login sapling's routing to be overridden.